### PR TITLE
record sample now populates input with code value

### PR DESCRIPTION
**Pull request by laclark22**
On the record sample form user can now click on a code from the record sample modal that pops up when code set button is clicked and that code will be populated in the input box associated with that value set. 

**Submitter:** Laura Clark
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR:
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:** Michael O'Keefe

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/views/checklist_tests/_checklist_measure.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 <% if !hide_patient_calculation? %>
   <div id='modify_record'><button class='btn btn-default' type='button' id="modifyrecord">Edit Test</button></div>
 <% end %>
-<% measures.sort_by(&:cms_int).each.with_index do |measure, measure_index| %>
+<% @measures.sort_by(&:cms_int).each do |measure, measure_index| %>
```
Comment (ljades) [inline app/views/checklist_tests/_checklist_measure.html.erb]: Any reason for the replacing the local variable with an instance variable? (`measures` to `@measures`)?

**Comment by laclark22** [comment]
So weird.. I have a record sample where there is only one element in the list and the modal closes. @ljades and @okeefm can you tell me which measure you guys are seeing this happen with?

**Comment by laclark22** [comment]
Fixed the modal not closing issue @okeefm and @ljades
